pkg/controller/certificaterequests/selfsigned/selfsigned_test.go

```diff
@@ -127,7 +128,19 @@ func TestSign(t *testing.T) {
 			corev1.TLSPrivateKeyKey: skECPEM,
 		},
 	}
-	csrECPEM := generateCSR(t, skEC, x509.ECDSAWithSHA256)
+	csrECPEM := generateCSR(t, skEC, x509.ECDSAWithSHA256, "test-ec")
+
+	emptyCertKeySecret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      rsaKeySecret.Name,
+			Namespace: gen.DefaultTestNamespace,
+		},
+		Data: map[string][]byte{
+			corev1.TLSPrivateKeyKey: skECPEM,
+		},
+	}
```

Comment:
Why is this added?

Reply:
I added it for the initial PR, copying existing tests while making the new one.

Good point though - this can totally use the existing EC secret through, so I'll use that.